README.md

```diff
@@ -92,14 +92,20 @@ scrape_configs:
         target_label: instance
       - target_label: __address__
         replacement: <<REDIS-EXPORTER-HOSTNAME>>:9121
+  ## it's also necessary
```

Comment:
I'd reword this, it's not necessary (for redis instance scraping to work) but probably a very good idea to scrape the exporter itself so how about `It's also highly recommended to scrape the exporter itself` ?

Reply:
It's OK,  thank you